src/main/java/com/wiley/driver/factory/capabilities/ChromeCaps.java

```diff
@@ -28,6 +30,11 @@ public ChromeOptions get() {
         ChromeOptions caps = getChromeOptions();
         if (!this.customCaps.asMap().isEmpty()) {
             caps.merge(this.customCaps);
+
+            // This is a workaround for ChromeOptions.merge() issue when passed arguments are not being assigned properly
+            Map chromeCapability = (Map) caps.getCapability(ChromeOptions.CAPABILITY);
+            List argumentsList = (List) chromeCapability.get("args");
+            argumentsList.forEach(o -> caps.addArguments(o.toString()));
```

Comment:
.get may return null and it will cause NPE on driver startup - again, veeery unlikely, it is set in caps initialization method, so worst case scenario - zero length list, which won't cause any troubles during iterate.

Reply:
I don't see where/how is it set (i looked into ChomeOptions and didn't find it quckly). Please provide a proof link

Reply:
Like I've mentioned above, caps variable is initialized with getChromeOptions() method (ChromeOptions caps = getChromeOptions();), which adds ChromeOptions with some pre-defined params (options.setCapability(ChromeOptions.CAPABILITY, options);)
Now this capability will always have an args field, which is empty by default but always initialized, so I can't see a way to get NPE with my changes after calling getChromeOptions().